
pad_map:

  square_button:   [3,  'digital']
  x_button:        [0,  'digital']
  circle_button:   [1,  'digital']
  triangle_button: [2,  'digital']
  L1_button:       [4,  'digital'] 
  R1_button:       [5,  'digital'] 
  share_button:    [8,  'digital']
  options_button:  [9,  'digital']
  L3_button:       [11, 'digital']
  R3_button:       [12, 'digital']
  ps4_button:      [10, 'digital']
  touchpad_button: [13, 'digital']

  horizontal_axis_stick_left:  [0,  'analog']
  vertical_axis_stick_left:    [1,  'analog']
  horizontal_axis_stick_right: [3,  'analog']
  L2_trigger:                  [2,  'analog']
  R2_trigger:                  [5,  'analog']
  vertical_axis_stick_right:   [4,  'analog']
  horizontal_cross_pad:        [6,  'analog']
  vertical_cross_pad:          [7, 'analog']
